### PR TITLE
Improve codebase terminology

### PR DESCRIPTION
**Pull request by wenovus**

### CODE REVIEW

File: docs/protobuf_getting_started.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
...
 
-* Where there is no native language binding generator for the language. Whilst [ygot](https://github.com/openconfig/ygot) and [pyangbind](https://github.com/robshakir/pyangbind) provide solutions for Go and Python respectively, there are limited toolkits for other languages. Use of a protobuf schema allows usable code artefacts to be generated through use of `protoc`. Protobuf supports a number of languages both [natively](https://developers.google.com/protocol-buffers/docs/reference/overview) and via [third-party plugins](https://github.com/google/protobuf/blob/master/docs/third_party.md).
+* Where there is no language binding generator for the language. Whilst [ygot](https://github.com/openconfig/ygot) and [pyangbind](https://github.com/robshakir/pyangbind) provide solutions for Go and Python respectively, there are limited toolkits for other languages. Use of a protobuf schema allows usable code artefacts to be generated through use of `protoc`. Protobuf supports a number of languages both [built-in](https://developers.google.com/protocol-buffers/docs/reference/overview) and via [third-party plugins](https://github.com/google/protobuf/blob/master/docs/third_party.md).
```
Comment (robshakir) [inline docs/protobuf_getting_started.md:7]: Should we say "where ygot does not support generating language bindings"?

File: ytypes/container.go [hunk 2]
```diff
@@ -253,7 +253,7 @@ func unmarshalStruct(schema *yang.Entry, parent interface{}, jsonTree map[string
 }
 
 // validateContainerSchema validates the given container type schema. This is a
-// sanity check validation rather than a comprehensive validation against the
+// quick check validation rather than a comprehensive validation against the
```
Comment (robshakir) [inline ytypes/container.go:256]: Just `this is a quick check`, no need for the word validation.

File: ytypes/bitset_type.go [hunk 3]
```diff
@@ -79,7 +79,7 @@ func validateBitsetSlice(schema *yang.Entry, value interface{}) error {
 	return nil
 }
 
-// validateBitsetSchema validates the given Bitset type schema. This is a sanity
+// validateBitsetSchema validates the given Bitset type schema. This is a quick
```
Comment (robshakir) [inline ytypes/bitset_type.go:82]: "This is a quick check", no need for the word validation.

File: ytypes/decimal_type.go [hunk 4]
```diff
@@ -72,7 +72,7 @@ func validateDecimalSlice(schema *yang.Entry, value interface{}) error {
 }
 
 // validateDecimalSchema validates the given decimal type schema. This is a
-// sanity check validation rather than a comprehensive validation against the
+// quick check validation rather than a comprehensive validation against the
```
Comment (robshakir) [inline ytypes/decimal_type.go:75]: Same as above.

File: ytypes/empty_type.go [hunk 5]
```diff
@@ -41,7 +41,7 @@ func validateEmpty(schema *yang.Entry, value interface{}) error {
 	return nil
 }
 
-// validateEmptySchema validates the given empty type schema. This is a sanity
+// validateEmptySchema validates the given empty type schema. This is a quick
```
Comment (robshakir) [inline ytypes/empty_type.go:44]: Same as above.

File: ytypes/int_type.go [hunk 6]
```diff
@@ -127,7 +127,7 @@ func validateIntSlice(schema *yang.Entry, value interface{}) error {
 	return nil
 }
 
-// validateIntSchema validates the given integer type schema. This is a sanity
+// validateIntSchema validates the given integer type schema. This is a quick
```
Comment (robshakir) [inline ytypes/int_type.go:130]: Same as above - throughout, it looks like this is a standard comment.

File: ytypes/util_schema.go [hunk 7]
```diff
@@ -33,7 +33,7 @@ func validateLengthSchema(schema *yang.Entry) error {
 		return nil
 	}
 	for _, r := range schema.Type.Length {
-		// This is a limited sanity check. It's assumed that a full check is
+		// This is a limited quick check. It's assumed that a full check is
```
Comment (robshakir) [inline ytypes/util_schema.go:36]: Just "limited check".

**Review by robshakir — CHANGES_REQUESTED** [review]
Thanks for cleaning these up.

File: docs/protobuf_getting_started.md (hunk 1 above)
Comment (wenovus) [reply docs/protobuf_getting_started.md:7]: The following sentence already mentions ygot and pyang. I changed the wording to "Where there is no **existing** language binding generator for the language"

File: ytypes/int_type.go (hunk 6 above)
Comment (wenovus) [reply ytypes/int_type.go:130]: I changed all places.

**Review by robshakir — APPROVED** [review]
Thanks -- LGTM!

**Pull request merged**